pgjdbc/src/main/java/org/postgresql/util/KerberosTicket.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2021, PostgreSQL Global Development Group
+ * See the LICENSE file in the project root for more information.
+ */
+
+package org.postgresql.util;
+
+import static org.postgresql.util.internal.Nullness.castNonNull;
+
+import org.postgresql.PGProperty;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+import javax.security.auth.Subject;
+import javax.security.auth.callback.Callback;
+import javax.security.auth.callback.CallbackHandler;
+import javax.security.auth.callback.UnsupportedCallbackException;
+import javax.security.auth.login.AppConfigurationEntry;
+import javax.security.auth.login.Configuration;
+import javax.security.auth.login.LoginContext;
+import javax.security.auth.login.LoginException;
+
+public class KerberosTicket {
+
+  private static final String CONFIG_ITEM_NAME = "pgjdbc";
```

Comment:
Well we are going to use it here https://github.com/pgjdbc/pgjdbc/blob/f61fbfe7b72ccf2ca0ac2e2c366230fdb93260e5/pgjdbc/src/main/java/org/postgresql/gss/MakeGSS.java#L38 to make the actual connection. So it seems appropriate to have the same name. I agree we could get away with just checking for the cache, but since we are going to the trouble, we might as well make sure it is going to work

Reply:
That's fine, but wanted to note it's a departure of how it used to work before.

Reply:
Ya, I am aware. Which is why I wanted someone else to look over it for sanity. I have to admit to being a bit in the dark so looking for advice from an expert.

Reply:
I'm not an expert in this area unfortunately, btw.

Reply:
hmmm... trial by fire then..